app/controllers/spree/orders_controller.rb

```diff
@@ -25,6 +25,7 @@ class OrdersController < ::BaseController
     before_action :check_at_least_one_line_item, only: :update
 
     def show
+      process_payment_intent!(params["payment_intent"])
```

Comment:
when does this action get visited within the entire subscription's workflow? sorry, I'm not following.

Reply:
We specify a redirect_url to Stripe so that after the customer visits Stripe in order to authorize the payment, they are redirected to that url. I chose the Orders#show url for the redirect.